lib/core/macros.nim

```diff
@@ -945,7 +945,7 @@ proc treeTraverse(n: NimNode; res: var string; level = 0; isLisp = false, indent
   case n.kind
   of nnkEmpty, nnkNilLit:
     discard # same as nil node in this representation
-  of nnkCharLit .. nnkInt64Lit:
+  of nnkCharLit .. nnkUInt64Lit:
     res.add(" " & $n.intVal)
```

Comment:
Not correct, you need nnkUInt64Lit as a special case that does `res.add(" " & $cast[uint64](n.intVal))`